library/core/src/lib.rs

```diff
@@ -110,8 +110,7 @@
 //
 // Library features:
 // tidy-alphabetical-start
-#![cfg_attr(bootstrap, feature(no_coverage))] // rust-lang/rust#84605
-#![cfg_attr(not(bootstrap), feature(coverage_attribute))] // rust-lang/rust#84605
+// rust-lang/rust#84605
```

Comment:
Shouldn't this comment still be attached to `coverage_attribute`? (now further below)

Reply:
I'm going to just delete this comment. I don't see why linking the tracking issue is adding value here.